cvpy/annotation/base/AnnotationLabel.py

```diff
@@ -19,13 +22,23 @@ def color(self) -> str:
     @color.setter
     def color(self, color) -> None:
         self._color = color
-        
+
     def as_dict(self) -> dict:
-        '''
+        """
         Creates a dictionary representation of this object.
         :return: A dictionary with all of the properties as keys and the property values as values
```

Comment:
I modified all docstrings to use the numpydoc convention such that it is consistent with other docstrings in the project. 

I used :return and :param because my editor autogenerated a docstring with those labels.